docs/gatsby-config.js

```diff
@@ -51,19 +57,17 @@ module.exports = {
             'caching/http-cache',
             'caching/troubleshooting',
           ],
-          Advanced: [
+          Networking: [
+            'advanced/interceptors-http',
+            'advanced/http-engine',
+            'advanced/no-runtime',
+          ],
+          'Advanced Config': [
```

Comment:
I'm not a huge fan of `Advanced Config` vs `Advanced Operations` sections as I'm not sure what the difference is. Also I feel we need to educate about is "operations" (in the GraphQL acceptation) and using it for things like "batching" or "upload" feels confusing.

Reply:
Ok! I'll revert this and we can keep 🤔ing